docs/examples/machine_translation/gnmt.md

```diff
@@ -1,6 +1,8 @@
-# Google Neural Machine Translation (GNMT)
+# Google Neural Machine Translation (GNMT) with Transformers
```

Comment:
You can rename this to "Machine Translation".
Please also change the filename from gnmt.md to translation.md in that case.

Reply:
Thanks for your advice. I have made changes accordingly.